sdk/lib/3rdparty/freetype/src/type1/t1gload.c

```diff
@@ -335,8 +338,8 @@
     if ( error )
 #ifdef __REACTOS__
     {
-        free(&decoder);
-        return error;
+      free(&decoder);
+      return error;
```

Comment:
Please keep the previous indentation (to reduce the diff)

Reply:
OK.